src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -439,6 +439,26 @@ def get_encryption_config(self, daemon_name=None, owner=None):
         return CephService.get_encryption_config(daemon_name)
 
 
+@UIRouter('/rgw/bucket', Scope.RGW)
+class RgwBucketUi(RgwBucket):
+    @Endpoint('GET')
+    @ReadPermission
+    def getTotalBucketsAndUsersLength(self, daemon_name=None):
```

Comment:
get is assumed from the endpoint itself (GET).
```suggestion
    def buckets_and_users_count(self, daemon_name: str =None):
```